driver/razerkbd_driver.c

```diff
@@ -1842,6 +1859,12 @@ static ssize_t razer_attr_write_matrix_effect_reactive(struct device *dev, struc
         request.transaction_id.id = 0x3F;
         break;
 
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_PRO_WIRELESS:
+        razer_set_device_mode(device, 0x00, 0x00);
```

Comment:
This should really not be triggered here. Does the effect really not work when the device is in mode 0x03-0x00 (driver mode)?

Reply:
Judge me if you want, but I got the keyboard wednesday. It wokr with `0x00, 0x00`, I was incorrectly testing (turning the option in polychromatic and waiting, not inputting anything)

Reply:
Or I did need `0x03, 0x00` ?

Reply:
Weird, it was working yesterday now it is not.

Reply:
Ok so `reactive` works without this call. But Ripple is still not?